src/modules/pageTable.js

```diff
@@ -41,36 +64,53 @@ class PageTableRow extends React.PureComponent {
     }
 
     _onClick() {
-        const { isClickable, row, rowProps } = this.props;
+        const { isClickable, onClick, row, rowProps } = this.props;
         const isTextSelect = window.getSelection().toString();
 
-        if (isClickable && !isTextSelect) {
+        if (_.isFunction(onClick)) {
```

Comment:
So no `isClickable` check for this case?  Basically, if the consumer says `isClickable={false}` _and_ binds an `onClick` handler function, you're assuming that the binding of the function indicates intention more than `isClickable={false}`? =)

Reply:
Restored as it was.